specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2019-01-01-preview/cpimTenant.json

```diff
@@ -0,0 +1,917 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "B2CTenants",
+    "version": "2019-01-01-preview",
```

Comment:
>"2019-01-01-preview", [](start = 15, length = 21)

Why are we adding 2 year old API version?

Reply:
Hi chiragg4u,

This is the version currently publicly exposed being used in our portal and elsewhere. With previous reviewers we discussed that it's ok to keep this version, to get it published, and consider a new version/date when this is GA'd.